_posts/2019-09-23-activate-azure.md

```diff
@@ -0,0 +1,78 @@
+author: shresth
+---
+### How to get the Github student developer pack to activate your Azure account
+
+![Ms Azure](https://www.tricentis.com/wp-content/uploads/2019/04/Microsoft-Azure-Logo-1024x753.png)
+
+ As a student, you can apply for the GitHub Student Developer Pack, which includes offers and benefits from GitHub partners.
+
+ You can use the Github student developer pack to activate your Microsoft Azure account so that you can access the benefits.
+
+ Below are the steps to be followed to get the student pack:
+
+ 1. Go to [Github Education](https://education.github.com "Github Education") and, in the top right, click on "Get benefits".
+ Select the benefits for students.
+ 2. Under "Which best describes your academic status", you need to select 'Student'.
+
+ ![status](https://help.github.com/assets/images/help/education/academic-status-student.png)
+ 3. Now you need to add the email address that you use for college.
+
+ ![email](https://help.github.com/assets/images/help/education/select-email-address.png)
+
+ 4. Then you need to upload a picture for proof of your academic status (college ID will do just fine). You can do this by two ways:
+
+  a) Use your camera to take a picture or
+
+  ![cam1](https://help.github.com/assets/images/help/education/upload-proof-status.png)
+
+  b) Upload the photo directly from your device
+
+  ![cam2](https://help.github.com/assets/images/help/education/upload-image-link.png)
+
+ 5. Next step is to add your college name.
+
+ ![collegename](https://help.github.com/assets/images/help/education/enter-school-name.png)
+ 6. The last step is to describe the purpose on how do you plan to use Github. You can tell about your willingness to contribute to the open source here.
+
+ ![contribute](https://help.github.com/assets/images/help/education/purpose-for-github-education.png)
+
+ Verify your application details, and click on "Submit your Information".
+
+ ![Submit](https://help.github.com/assets/images/help/education/submit-your-information-button.png)
+
+
+  **You'll get the confirmation shortly.**
+
+![Azure1](https://github.com/shresth26/glugmvit.github.io/blob/Azure-patch-1/assets/images/Azure/Azure.png)
+
+
+  <h3>How to activate Azure from your pack</h3>
+
+
+
+  Once you get the confirmation email for your student developer pack(this may come on the same day or may take 10 days depending on your luck) you need to go back to the Github website using this link [Offers](https://education.github.com/pack/offers) and avail the benefits.
+
+  There will be many tools available so you need to scroll down and find Microsoft Azure. Click to generate your unique code.
+
+  ![Azure2](https://github.com/shresth26/glugmvit.github.io/blob/Azure-patch-1/assets/images/Azure/Azure2.png)
+
+  Now go to the Microsoft website and click on "Activate Now" button where you'll get a prompt to sign in.
+
+  ![Azure3](https://github.com/shresth26/glugmvit.github.io/blob/Azure-patch-1/assets/images/Azure/Azure3.png)

```

Comment:
Specify a relative link for the images uploaded to the repo, not an absolute link.

Example: ![Azure3](/assets/images/Azure/Azure3.png)